tests/api/utils/test_definition_rendering.py

```diff
@@ -83,6 +85,86 @@ def test_no_resolve_in_version_1():
         render_definition_template(definition, {})
 
 
+@mock.patch.dict(os.environ, {"A": "value"}, clear=True)
+@mock.patch(f"{NATIVEAPP_MODULE}.cc.warning")
+def test_no_resolve_and_warning_in_version_1(warning_mock):
+    definition = {
+        "definition_version": "1",
+        "native_app": {"name": "test_source_<% ctx.env.A %>", "artifacts": []},
+    }
+    result = render_definition_template(definition, {}).project_context
+
+    assert result == {
+        "ctx": {
+            "definition_version": "1",
+            "native_app": {"name": "test_source_<% ctx.env.A %>", "artifacts": []},
+            "env": ProjectEnvironment({}, {}),
+        }
+    }
+    warning_mock.assert_called_once_with(
+        "Ignoring template pattern in 'snowflake.yml' project definition file. "
+        "Update definition_version to 1.1 or later to enable template expansion."
```

Comment:
In this context snowflake.yml vs snowflake.local.yml does matter though. If you're saying something like "update the version in your snowflake.yml", it's OK to assume the user understands override. If you say "I found templates in file 'snowflake.yml'" but it's really "snowflake.local.yml", then it's misleading, as the name is expected to be a specific file on the FS. If you can't get the specific file name without significant surgery, I'd do something like:

```
Ignoring template pattern in project definition file. Update 'definition_version' to 1.1 or later in snowflake.yml to enable template expansion.
```

Reply:
updated